src/transformations/execute.ts

```diff
@@ -232,6 +236,47 @@ export async function executeTransformations(
         vimState.editor.selection = new vscode.Selection(newPos, newPos);
         break;
 
+      case 'executeNormal':
+        const keystroke = keystrokesExpressionParser.parse(transformation.keystroke);
+        const startLineNumber = transformation.startLineNumber;
+        const endLineNumber = transformation.endLineNumber;
+        const withRange = transformation.withRange;
```

Comment:
Make this one destructuring assignment

Reply:
Fix it.
[9d4d457](https://github.com/VSCodeVim/Vim/pull/8896/commits/9d4d4575b6f8f7d9da4f3d1765583d22cac97c58)